mobile/lib/features/wallet/application/util.dart

```diff
@@ -3,3 +3,37 @@ String truncateWithEllipsis(int cutoff, String text) {
       ? text
       : '${text.substring(0, (cutoff / 2).floor())}...${text.substring(text.length - (cutoff / 2).ceil(), text.length)}';
 }
+
+// Formats any number less than 10 bitcoin in the following format
```

Comment:
I believe doc comments have 3 `/`.

Reply:
Also in dart? 🤔

Reply:
Yap :) https://dart.dev/effective-dart/documentation

Reply:
Next time then 🙂